src/API/useInAppUpload.js

```diff
@@ -2,17 +2,31 @@ import { useMutation } from 'react-query';
 
 import { useOkapiKy } from '@folio/stripes/core';
 
+import { CAPABILITIES_VALUE } from '../constants';
+
 export const useInAppUpload = () => {
   const ky = useOkapiKy();
 
-  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates }) => {
-    return ky.post(`bulk-edit/${jobId}/item-content-update/upload`, {
-      searchParams: { limit: 10 },
-      json: {
-        itemContentUpdates: contentUpdates,
-        totalRecords: contentUpdates.length,
-      },
-    }).json();
+  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates, capability }) => {
+      const typeOfBulk = CAPABILITIES_VALUE[capability];
```

Comment:
what's wrong with your eslint? you have a mix of 4 and 2 indents

Reply:
@NikitaSedyx what do you mean here? I can't understand what's wrong here? `const typeOfBulk `is placed inside the `useMutation.` Lint showing that it's okay here

Reply:
I mean that you have issues with codestyle